### PR TITLE
5.2 task solution

### PR DESCRIPTION
**Pull request by KovalAlexey**

### CODE REVIEW

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -7,5 +7,80 @@
...
+
+        public PlaneBuilder setPlaneID(int planeID) {
+            this.planeID = planeID;
+
+            return this;
+        }
+
+        public PlaneBuilder setPlaneName(String planeName) {
+            this.planeName = planeName;
+
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Plane.java]: redundant empty line

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -7,5 +7,80 @@
...
+        private String pilotName;
+        private int gasCapacity;
+
+        private PlaneBuilder() {
+
+        }
+
+        public PlaneBuilder setPlaneID(int planeID) {
+            this.planeID = planeID;
+
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Plane.java]: redundant empty line

File: src/main/java/core/basesyntax/Plane.java [hunk 3]
```diff
@@ -7,5 +7,80 @@
...
+        public PlaneBuilder setPilotName(String pilotName) {
+            this.pilotName = pilotName;
+
+            return this;
+        }
+
+        public PlaneBuilder setGasCapacity(int gasCapacity) {
+            this.gasCapacity = gasCapacity;
+
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Plane.java]: a redundant empty line. Please, check all your code

File: src/main/java/core/basesyntax/Plane.java [hunk 4]
```diff
@@ -7,5 +7,80 @@
...
+    }
+
+    public static class PlaneBuilder {
+        private int planeID;
+        private String planeName;
+        private boolean isInAir;
+        private String pilotName;
+        private int gasCapacity;
+
+        private PlaneBuilder() {
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Plane.java]: please add the Main class with an example of how to use your builder

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Plane.java [hunk 5]
```diff
@@ -7,5 +7,77 @@
...
+    }
+
+    public static class PlaneBuilder {
+        private int planeID;
+        private String planeName;
+        private boolean isInAir;
+        private String pilotName;
+        private int gasCapacity;
+
+        public PlaneBuilder() {
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Plane.java]: is it required? By default, we already have a default constructor

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Plane.java [hunk 6]
```diff
@@ -7,5 +7,74 @@
  */
 
 public class Plane {
+    private int planeID;
+    private String planeName;
+    private boolean isInAir;
+    private String pilotName;
+    private int gasCapacity;
 
+    public Plane(PlaneBuilder planeBuilder) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Plane.java]: https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_05.html
check your solution according to checklist

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**